CMakeLists.txt

```diff
@@ -161,10 +161,18 @@ option(dht "enable support for Mainline DHT" ON)
 option(resolve-countries "enable support for resolving countries from peer IPs" ON)
 option(unicode "enable unicode support" ON)
 option(deprecated-functions "enable deprecated functions for backwards compatibility" ON)
+if (NOT MSVC)
+	# MSVC doesn't provide this option and has its own ABI issues anyway.
+	option(abi "pick one until libtorrent's ABI is robust: c++03, c++11, c++14, default (currently c++03)" "default")
```

Comment:
That's actually a very good question, and I might have to check if this is actually still relevant now that asio is built header-only. Let me get back to you on this.